etl/steps/data/grapher/wid/2023-01-27/world_inequality_database.py

```diff
@@ -0,0 +1,35 @@
+"""Load a garden dataset and create a grapher dataset."""
+
+from owid.catalog import Dataset
+
+from etl.helpers import PathFinder
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load garden dataset.
+    ds_garden: Dataset = paths.load_dependency("world_inequality_database")
+
+    # Read table from garden dataset.
+    tb_garden = ds_garden["world_inequality_database"]
+
+    #
+    # Process data.
+    #
+
+    #
+    # Save outputs.
+    #
+    # Create a new grapher dataset with the same metadata as the garden dataset.
+    ds_grapher = Dataset.create_empty(dest_dir, ds_garden.metadata)
```

Comment:
Here you could do `ds_grapher = create_dataset(dest_dir, tables=[tb_garden], default_metadata=ds_garden.metadata)` and delete the next line of code.